pcap-linux.c

```diff
@@ -206,6 +206,11 @@
 # endif /* PCAP_SUPPORT_PACKET_RING */
 #endif /* PF_PACKET */
 
+ /* check if kernel supports fanout for socket */
+# ifdef PACKET_FANOUT
+#  define HAVE_FANOUT
```

Comment:
As this is a Linux-only file, wouldn't it be sufficient to just test `PACKET_FANOUT` in the `#ifdef`s, rather than defining a new symbol?